archetypes/resources.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "{{ replaceRE "^[-0-9]{10}-" "" .Name | replaceRE "-" " " | title }}"
+slug: "{{ replaceRE "^[-0-9]{10}-" "" .Name }}"
+draft: false
+event_date: "Write the date of the event"
+image: "event-image.png"
+name: "Write the name or title of the event"
+description: "Write the event's description"
+events: ['Podcast', 'Webinar', 'Conference Talk']
```

Comment:
Using `resources` as a taxonomy term would've created a `/resources` page for all of the resources. It would be better to list all of the resources on the `/resources` page. 

In order to prevent confusion, I thought that using `events` for `event_type` would be a little better while eliminating the issue with Hugo not displaying the correct page. If you think another term may be more specific, let me know.